File: Android/java/doraemonkit/src/main/java/com/didichuxing/doraemonkit/config/CrashCaptureConfig.java
package com.didichuxing.doraemonkit.config;

import com.didichuxing.doraemonkit.constant.SharedPrefsKey;
import com.didichuxing.doraemonkit.util.DoKitSPUtil;

public class CrashCaptureConfig {
    public static boolean isCrashCaptureOpen() {
        return DoKitSPUtil.getBoolean(SharedPrefsKey.CRASH_CAPTURE_OPEN, false);
    }

    public static void setCrashCaptureOpen(boolean open) {
        DoKitSPUtil.putBoolean(SharedPrefsKey.CRASH_CAPTURE_OPEN, open);
    }
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/config/CrashCaptureConfig.kt
package com.didichuxing.doraemonkit.config

import com.didichuxing.doraemonkit.constant.SharedPrefsKey
import com.didichuxing.doraemonkit.util.SharedPrefsUtil

/**
 * Created by wangxueying on 2020-06-30.
 */
object CrashCaptureConfig {
    var isCrashCaptureOpen: Boolean
        get() = SharedPrefsUtil.getBoolean(SharedPrefsKey.CRASH_CAPTURE_OPEN, false)
        set(open) = SharedPrefsUtil.putBoolean(SharedPrefsKey.CRASH_CAPTURE_OPEN, open)
}
